### PR TITLE
feat: func StringToSymbolCode

### PR DESCRIPTION
**Pull request by lockp111**

### CODE REVIEW

File: types.go [hunk 1]
```diff
@@ -227,6 +227,26 @@ type Symbol struct {
 
 type SymbolCode uint64
 
+func StringToSymbolCode(str string) (SymbolCode, error) {
```
Comment (maoueh) [inline types.go]: A few tests would be needed before merging that, are ok adding them?

Comment (lockp111) [reply types.go]: I don't know how to add the test, I deployed a contract to verify it on EOS.

**Comment by abourget** [comment]
There's symbols, symbols+precision and assets. Right now, we decide the Asset and directly unpack the precision+symbol.. but it'd be really useful to decode the symbol+precision alone too.

We even need it for the state database @maoueh .. let's wrap that up and merge it somehow.

**Comment by maoueh** [comment]
Replaced by #121 which added tests for the `StringToSymbolCode` function. Added `ToName` at the same time on `SymbolCode` to make it easier to transform it right away to name encoded value.

**Pull request closed without merging**